src/templates/app/user/security/account_closure.jsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import Loading from '../../../_common/components/loading.jsx';
+import { SeparatorLine } from '../../../_common/components/separator_line.jsx';
+import { FormRow, SubmitButton } from '../../../_common/components/forms.jsx';
+
+const AccountClosure = () => (
+    <React.Fragment>
+        <div id='msg_main' className='center-text gr-gutter gr-padding-10 invisible'>
+            <h1>{it.L('Account closure confirmed')}</h1>
+            <p className='notice-msg'>{it.L('Accounts closed successfully. A confirmation email will be sent to your email.[_1]This page will redirect to the [_2] homepage after 10 seconds.', '<br />', it.website_name)}</p>
+        </div>
+        <div id='closure-container'>
+            <div id='main-header' className='gr-padding-30'>
+                <h1 id='heading'>{it.L('Account Closure')}</h1>
+                <p>{it.L('Closing your [_1] accounts involves closing all open positions in your accounts, and withdrawing your funds, and deactivating your accounts with [_1]', it.website_name)}</p>
+            </div>
+
+            <div className='gr-no-gutter' id='closure_description'>
+                <h2 className='primary-color'>{it.L('What would you like to do?')}</h2>
+                <fieldset>
+                    <div className='gr-padding-20 gr-gutter-left gr-gutter-right'>
+                        <ClosureDescription
+                            title={it.L('Create crypto account')}
+                            subtitle={it.L('[_1]Open an account[_2] in the cryptocurrency of your choice:', `<a href="${it.url_for('new_account/realws')}">`, '</a>')}
+                            list_items={[
+                                it.L('Bitcoin (BTC)'),
+                                it.L('Ether (ETH)'),
+                                it.L('Litecoin (LTC)'),
+                                it.L('Tether (UST)'),
+                            ]}
+                        />
+                        <ClosureDescription
+                            title={it.L('Change my affiliate')}
+                            subtitle={it.L('Contact [_1]affiliates@binary.com[_2] for more info on changing your affiliate.', '<a href="mailto:affiliates@binary.com">', '</a>')}
+                        />
+                        <ClosureDescription
+                            title={it.L('Change my account limits')}
+                            subtitle={it.L('You may set limits in your account to help prevent unwanted losses.[_1]Go to [_2]self-exclusion page[_3] to manage your account limits.', '<br />', `<a href=${it.url_for('user/security/self_exclusionws')}>`, '</a>')}
```

Comment:
same here, please separate into two strings and don't use `<br/>` in the translation string. for example, change `subtitle` to accept an array of items and split them by `<br/>` in the component.